sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/implementation/serializer/Iso8601DateDeserializer.java

```diff
@@ -14,12 +14,12 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
-class Iso8601DateDeserializer extends UntypedObjectDeserializer {
+public final class Iso8601DateDeserializer extends UntypedObjectDeserializer {
```

Comment:
Do we need to make this public?

Reply:
I moved what uses this into another implementation package